src/app/hooks/useFetch.ts

```diff
@@ -11,7 +11,11 @@ export default function useFetch<T>(url: string): {
   const isLoading = data === null;
 
   function refetch() {
-    fetch(url)
+    fetch(url, {
+      headers: {
+        Authorization: 'e63f4351-b625ddac-254af606-5a2d8ef0',
```

Comment:
This should not be included in the useFetch hook, also this looks like a secret.